guides/common/modules/proc_estimating-the-size-of-a-backup.adoc

```diff
@@ -17,6 +17,7 @@ A full backup requires space to store the following data:
 .Procedure
 . Enter the `du` command to estimate the size of uncompressed directories containing {Project} database and configuration files:
 +
+.For RHEL 7:
```

Comment:
We started a convention to use `{EL}` as a macro. For Satellite this expands to `{RHEL}` which expands to the full Red Hat Enterprise Linux. In upstream it expands to Enterprise Linux.